packages/breadcrumbs/src/BreadcrumbItem.ts

```diff
@@ -86,7 +86,9 @@ export class BreadcrumbItem extends LikeAnchor(Focusable) {
         return html`
             <a
                 id="item-link"
-                href=${ifDefined(!this.isLastOfType ? this.href : undefined)}
+                href=${ifDefined(
+                    !this.isLastOfType ? this.href || '#' : undefined
```

Comment:
Could this lead to a jump to the top of the page when `"#"` is set?

Reply:
Good catch! I have updated the changes. Let' see if we can land this for tomorrow's release.